### PR TITLE
Added helper files as release assets and wget added to main script

### PR DESCRIPTION
**Pull request by cindydeng1998**
- Draft release https://github.com/Azure/iot-edge-config/releases/tag/untagged-546b16fd9f2957167ff7

### CODE REVIEW

File: src/azure-iot-edge-installer.sh [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
 echo "Running azure-iot-edge-installer.sh"
 
+# if helper scripts dont exist, fetch via wget 
+if [ -d "iot-edge-installer" ] 
+then
+    echo "Directory iot-edge-installer exists." 
+else
+    mkdir iot-edge-installer
+    cd iot-edge-installer
+    wget https://github.com/Azure/iot-edge-config/releases/latest/download/validate-tier1-os.sh
```
Comment (marianan) [inline src/azure-iot-edge-installer.sh:11]: >latest [](http://example.com/codeflow?start=59&length=6)

should the script pull the version of the components that map to its own version?

File: src/azure-iot-edge-installer.sh [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+    wget https://github.com/Azure/iot-edge-config/releases/latest/download/install-container-management.sh
+    wget https://github.com/Azure/iot-edge-config/releases/latest/download/install-edge-runtime.sh
+    wget https://github.com/Azure/iot-edge-config/releases/latest/download/validate-post-install.sh
+    echo "Downloaded helper files to temporary directory /iot-edge-installer"
```
Comment (mhshami01) [inline src/azure-iot-edge-installer.sh:15]: nit - should be './iot-edge-installer'

File: src/azure-iot-edge-installer.sh [hunk 3]
```diff
@@ -12,3 +26,11 @@ chmod +x validate-post-install.sh
 ./install-container-management.sh
 ./install-edge-runtime.sh
 ./validate-post-install.sh
+cd ..
```
Comment (mhshami01) [inline src/azure-iot-edge-installer.sh:29]: might be better (safer?) to store the `cwd` in a local variable and cd to $(local variable) instead?

File: vsts_ci/azure-pipelines.yml [hunk 4]
```diff
@@ -35,19 +35,19 @@ stages:
     steps:
     - script: |  
         mkdir dest
-        cp src/azure-iot-edge-installer.sh dest/ 
+        cp src/* dest/
```
Comment (mhshami01) [inline vsts_ci/azure-pipelines.yml:38]: is it better to use '*.sh'?

**Review by marianan — APPROVED** [review]
:shipit:

**Pull request merged**